src/components/CardContainer/CardContainer.tsx

```diff
@@ -4,21 +4,21 @@ import {ApiCard} from '../ApiCard/ApiCard'
 import './CardContainer.css'
 
 type ApiProp = {
-  apiList: Api[]
+  apiList: Api[],
+  toggleFavorite: (ApiCard: Api) => void,
+  favorites: Api[]
 }
-export const CardContainer = ({apiList}: ApiProp) => {
+export const CardContainer = ({apiList, toggleFavorite, favorites}: ApiProp) => {
 
   const ApiCards = apiList.map((apiObj, index) => {
-    return <ApiCard
-    API={apiObj.API}
-    Auth={apiObj.Auth}
-    Category={apiObj.Category}
-    Cors={apiObj.Cors}
-    Description={apiObj.Description}
-    HTTPS={apiObj.HTTPS}
-    Link={apiObj.Link}
-    key={index}
-    />
+    return (
+      <ApiCard
+        apiData={apiObj}
+        key={index}
+        toggleFavorite={toggleFavorite}
+        favorites={favorites}
+      />
+    )
```

Comment:
Noticed a slight bug with the favoriting toggle. When there is only one favorited card in localStorage, clicking on the favorite heart does not remove it. In other words I can unfavorite all but one of my favorited cards.